core/templates/pages/preferences-page/modal-templates/edit-profile-picture-modal.component.ts

```diff
@@ -30,12 +30,14 @@ require('cropperjs/dist/cropper.min.css');
   templateUrl: './edit-profile-picture-modal.component.html'
 })
 export class EditProfilePictureModalComponent extends ConfirmOrCancelModal {
-  uploadedImage: SafeResourceUrl;
+  // 'uploadedImage' will be null if the uploaded svg is invalid or not trusted.
+  uploadedImage: SafeResourceUrl | null = null;
   cropppedImageDataUrl: string = '';
   invalidImageWarningIsShown: boolean = false;
   allowedImageFormats: readonly string[] = AppConstants.ALLOWED_IMAGE_FORMATS;
-  cropper;
-  @ViewChild('croppableImage') croppableImageRef: ElementRef;
+  // 'cropper' will be undefined if it has not been initialized.
+  cropper: Cropper | undefined;
```

Comment:
Can we do non-null assertion here ?

Reply:
But you catered it with a if check, right ? Making it ` | undefined` would mean that we're assigning an undefined value (`this.cropper = undefined`) to it somewhere, which is not the case here.

Reply:
updated